roxie/roxiemem/roxiemem.cpp

```diff
@@ -8716,9 +8716,9 @@ static int compareTiming(const void * pLeft, const void * pRight)
 }
 
 
-class RoxieMemTimingTests2 : public CppUnit::TestFixture
```

Comment:
Not sure I understand why this name change was made

Reply:
The name was changed because the filter for running a unit test matches using a substring.  If the 2 is on the end of the test name it is impossible to only run the first test - since it will also run this test!

Reply:
Hmm. I would have gone for calling them tests1 and tests2 then...